frontend/src/pages/HomePage.tsx

```diff
@@ -1,13 +1,21 @@
 import React, {ReactElement} from 'react'
 import SearchBar from '../components/Home/SearchBar'
 import NavBar from '../components/Home/NavBar'
-
-
+import SectionDivider from '../components/Home/SectionDivider'
+import ApartmentCard from '../components/Home/ApartmentCard'
 
 const HomePage= (): ReactElement => {
     return <div className='Home'>
         <NavBar />   
-        <SearchBar />       
+        <SearchBar />   
+        <SectionDivider /> 
+        <div className="container-fluid">
+            <div className="row flex-row flex-nowrap pt-3">
+                <div className="col-4"><ApartmentCard address="117 Eddy St" company="Ithaca Renting Company" bedsAndBaths="5 Br | 2 B" amenities={false} price="$800" percentReview="90%" numReviews="5 Reviews"/></div> 
```

Comment:
`react-bootstrap` is slightly different from doing bootstrap in vanilla JS. We shouldn't be using `<div>`s here, but should rather be using the column / grid components provided by the `react-bootstrap` library. See this link for how to do that: https://react-bootstrap.github.io/layout/grid/

In doing this, you should set each of these cards to be 12 columns wide (entire row) for mobile or small (`sm`) screens, and 4 columns for medium and up.